### DISCUSSION TITLE
is there a way to use letters instead of numbers in markdown list?

### DISCUSSION THREAD

**Discussion by rczerminski-valo** [fonsp/Pluto.jl#1588] [Questions about using Pluto]
Is there a way to use letters instead of numbers in markdown list in Pluto?

So, instead of:
1. item1
2. item2

I would have:

a. item1
b. item2

**Comment by MichaelHatherly** [comment]
There's possibly a way to do this with the standard `md"` and some careful CSS/JS. Alternatively you use `CommonMark`s attribute syntax to set the `ol` `type = "a"` as follows which will be about as straightforward as you can get, though does require you to import a separate package for it:

![image](https://user-images.githubusercontent.com/6144086/138279683-30e6cdfc-1e9b-4c1c-a26e-44d2d9bb5b6f.png)

**Reply by rczerminski-valo** [reply]
This looks great!

Is there a way to use latex style snippets as below?
These work fine with md, but apparently do not work with cm.

cm"""
## Test CommonMark

This does work
{type="a"}
1. first
1. second

but this does not and err msg is: UndefVarError: n_s not defined

{type="A"}
1. try latex (e.g. $n_s = 100$) and $x$
1. fit model response surface: $model = fit(x, f_{obj}(x))$

It works fine with md
"""

**Reply by MichaelHatherly** [reply]
You'll need to use double backticks ` ``n_s`` ` when you want to write math/latex. The `cm"` syntax purposefully does not support using `$` syntax for that since it conflicts with Julia's string interpolation syntax. There's a lot of edge cases with `md"` and `$` syntax that have caused issues for a number of years, so I took the decision with `CommonMark` to remove the possibility of conflicts between them.

![image](https://user-images.githubusercontent.com/6144086/138285301-3e5cc12d-9a2f-48ee-b663-8a285b82c4db.png)

**Reply by rczerminski-valo** [reply]
Works great - thank you!

**Comment by Pangoraw** [comment]
You can also create a new css rule setting the [`list-style`](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style) property of ordered lists (ol) elements created by md:

![image](https://user-images.githubusercontent.com/9824244/138279935-79d8d081-2f55-445e-b328-2e2027308b6a.png)

**Reply by MichaelHatherly** [reply]
Any way to easily target a single list with that approach rather than changing them globally?

**Reply by Pangoraw** [reply]
The CSS selector might become a bit long and not very practical. Common mark seems a lot easier in this case.

**Reply by dralletje** [reply]
I'm looking at bringing [declarative shadow DOM](https://web.dev/declarative-shadow-dom/#detection-support) which, though conceptually a bit heavy, would allow stuff like this without the possibility of class collisions even.

**Reply by fonsp** [reply]
(@dralletje did that, released in 0.17.2, documentation Coming Soon™️)
